android.view.InflateException: Binary XML file line #19: Error inflating class com.devbrackets.android.exomedia.ui.widget.EMVideoView

- [ ] I have verified there are no duplicate active or recent bugs, questions, or requests

###### Include the following:
 - ExoMedia version: `3.0.0`
 - Device OS version: `4.4.2`
 - Devide Manufacturer: `LG`
 - Device Name: `LG G2`
 -PHONE_MODEL: `LG-D802`
 
###### Reproduction Steps
 0.  play a video in LG G2 using exomedia

###### Expected Result
exomedia play the video.
###### Actual Result
exomedia throws an unhandled the exception. I have been able to produce this only in LG G2. on other
devices works fine.


###### Full Stacktrace

> Unable to start activity ComponentInfo{com.rashavas.appido.v069.release/com.milad.appido.ExoMedia.VideoViewActitivity}: 
> android.view.InflateException: Binary XML file line #19: Error inflating class com.devbrackets.android.exomedia.ui.widget.EMVideoView
> dalvik.system.NativeStart in main
> com.android.internal.os.ZygoteInit in main at line 608
> com.android.internal.os.ZygoteInit$MethodAndArgsCaller in run at line 792
> java.lang.reflect.Method in invoke at line 515
> java.lang.reflect.Method in invokeNative
> android.app.ActivityThread in main at line 5105
> android.os.Looper in loop at line 136
> android.os.Handler in dispatchMessage at line 102
> android.app.ActivityThread$H in handleMessage at line 1200
> android.app.ActivityThread in access$800 at line 139
> android.app.ActivityThread in handleLaunchActivity at line 2250
> android.app.ActivityThread in performLaunchActivity at line 2164
> android.app.Instrumentation in callActivityOnCreate at line 1087
> android.app.Activity in performCreate at line 5275
> com.milad.appido.ExoMedia.VideoViewActitivity in onCreate at line 55
> android.app.Activity in setContentView at line 1973
> com.android.internal.policy.impl.PhoneWindow in setContentView at line 315
> android.view.LayoutInflater in inflate at line 353
> android.view.LayoutInflater in inflate at line 397
> android.view.LayoutInflater in inflate at line 492
> android.view.LayoutInflater in rInflate at line 745
> android.view.LayoutInflater in parseInclude at line 839
> android.view.LayoutInflater in rInflate at line 755
> android.view.LayoutInflater in createViewFromTag at line 696
> android.view.LayoutInflater in createView at line 594
> java.lang.reflect.Constructor in newInstance at line 423
> java.lang.reflect.Constructor in constructNative
> com.devbrackets.android.exomedia.ui.widget.EMVideoView in <init> at line 100
> com.devbrackets.android.exomedia.ui.widget.EMVideoView in setup at line 614
> com.devbrackets.android.exomedia.ui.widget.EMVideoView in readAttributes at line 636
> com.devbrackets.android.exomedia.ui.widget.VideoControlsMobile in <init> at line 49
> com.devbrackets.android.exomedia.ui.widget.VideoControls in <init> at line 166
> com.devbrackets.android.exomedia.ui.widget.VideoControls in setup at line 614
> android.view.View in inflate at line 17516
> android.view.LayoutInflater in inflate at line 353
> android.view.LayoutInflater in inflate at line 397
> android.view.LayoutInflater in inflate at line 492
> android.view.LayoutInflater in rInflate at line 758
> android.view.LayoutInflater in rInflate at line 758
> android.view.LayoutInflater in rInflate at line 755
> android.view.LayoutInflater in createViewFromTag at line 694
> android.view.LayoutInflater in onCreateView at line 669
> com.android.internal.policy.impl.PhoneLayoutInflater in onCreateView at line 56
> android.view.LayoutInflater in createView at line 594
> java.lang.reflect.Constructor in newInstance at line 423
> java.lang.reflect.Constructor in constructNative
> android.widget.ImageButton in <init> at line 83
> android.widget.ImageButton in <init> at line 87
> android.widget.ImageView in <init> at line 129
> android.content.res.TypedArray in getDrawable at line 602
> android.content.res.Resources in loadDrawable at line 2132
> android.graphics.drawable.Drawable in createFromXml at line 877
> android.graphics.drawable.Drawable in createFromXmlInner at line 933
> Caused by: org.xmlpull.v1.XmlPullParserException ("Binary XML file line #1: invalid drawable tag vector")
> dalvik.system.NativeStart in main
> com.android.internal.os.ZygoteInit in main at line 608
> com.android.internal.os.ZygoteInit$MethodAndArgsCaller in run at line 792
> java.lang.reflect.Method in invoke at line 515
> java.lang.reflect.Method in invokeNative
> android.app.ActivityThread in main at line 5105
> android.os.Looper in loop at line 136
> android.os.Handler in dispatchMessage at line 102
> android.app.ActivityThread$H in handleMessage at line 1200
> android.app.ActivityThread in access$800 at line 139
> android.app.ActivityThread in handleLaunchActivity at line 2250
> android.app.ActivityThread in performLaunchActivity at line 2164
> android.app.Instrumentation in callActivityOnCreate at line 1087
> android.app.Activity in performCreate at line 5275
> com.milad.appido.ExoMedia.VideoViewActitivity in onCreate at line 55
> android.app.Activity in setContentView at line 1973
> com.android.internal.policy.impl.PhoneWindow in setContentView at line 315
> android.view.LayoutInflater in inflate at line 353
> android.view.LayoutInflater in inflate at line 397
> android.view.LayoutInflater in inflate at line 492
> android.view.LayoutInflater in rInflate at line 745
> android.view.LayoutInflater in parseInclude at line 839
> android.view.LayoutInflater in rInflate at line 755
> android.view.LayoutInflater in createViewFromTag at line 696
> android.view.LayoutInflater in createView at line 594
> java.lang.reflect.Constructor in newInstance at line 423
> java.lang.reflect.Constructor in constructNative
> com.devbrackets.android.exomedia.ui.widget.EMVideoView in <init> at line 100
> com.devbrackets.android.exomedia.ui.widget.EMVideoView in setup at line 614
> com.devbrackets.android.exomedia.ui.widget.EMVideoView in readAttributes at line 636
> com.devbrackets.android.exomedia.ui.widget.VideoControlsMobile in <init> at line 49
> com.devbrackets.android.exomedia.ui.widget.VideoControls in <init> at line 166
> com.devbrackets.android.exomedia.ui.widget.VideoControls in setup at line 614
> android.view.View in inflate at line 17516
> android.view.LayoutInflater in inflate at line 353
> android.view.LayoutInflater in inflate at line 397
> android.view.LayoutInflater in inflate at line 492
> android.view.LayoutInflater in rInflate at line 758
> android.view.LayoutInflater in rInflate at line 758
> android.view.LayoutInflater in rInflate at line 755
> android.view.LayoutInflater in createViewFromTag at line 694
> android.view.LayoutInflater in onCreateView at line 669
> com.android.internal.policy.impl.PhoneLayoutInflater in onCreateView at line 56
> android.view.LayoutInflater in createView at line 594
> java.lang.reflect.Constructor in newInstance at line 423
> java.lang.reflect.Constructor in constructNative
> android.widget.ImageButton in <init> at line 83
> android.widget.ImageButton in <init> at line 87
> android.widget.ImageView in <init> at line 129
> android.content.res.TypedArray in getDrawable at line 602
> android.content.res.Resources in loadDrawable at line 2136
> Caused by: android.content.res.Resources$NotFoundException ("File res/drawable-xxhdpi-v4/exomedia_ic_skip_previous_white.xml from drawable resource ID #0x7f02006b")
> dalvik.system.NativeStart in main
> com.android.internal.os.ZygoteInit in main at line 608
> com.android.internal.os.ZygoteInit$MethodAndArgsCaller in run at line 792
> java.lang.reflect.Method in invoke at line 515
> java.lang.reflect.Method in invokeNative
> android.app.ActivityThread in main at line 5105
> android.os.Looper in loop at line 136
> android.os.Handler in dispatchMessage at line 102
> android.app.ActivityThread$H in handleMessage at line 1200
> android.app.ActivityThread in access$800 at line 139
> android.app.ActivityThread in handleLaunchActivity at line 2250
> android.app.ActivityThread in performLaunchActivity at line 2164
> android.app.Instrumentation in callActivityOnCreate at line 1087
> android.app.Activity in performCreate at line 5275
> com.milad.appido.ExoMedia.VideoViewActitivity in onCreate at line 55
> android.app.Activity in setContentView at line 1973
> com.android.internal.policy.impl.PhoneWindow in setContentView at line 315
> android.view.LayoutInflater in inflate at line 353
> android.view.LayoutInflater in inflate at line 397
> android.view.LayoutInflater in inflate at line 492
> android.view.LayoutInflater in rInflate at line 745
> android.view.LayoutInflater in parseInclude at line 839
> android.view.LayoutInflater in rInflate at line 755
> android.view.LayoutInflater in createViewFromTag at line 696
> android.view.LayoutInflater in createView at line 594
> java.lang.reflect.Constructor in newInstance at line 423
> java.lang.reflect.Constructor in constructNative
> com.devbrackets.android.exomedia.ui.widget.EMVideoView in <init> at line 100
> com.devbrackets.android.exomedia.ui.widget.EMVideoView in setup at line 614
> com.devbrackets.android.exomedia.ui.widget.EMVideoView in readAttributes at line 636
> com.devbrackets.android.exomedia.ui.widget.VideoControlsMobile in <init> at line 49
> com.devbrackets.android.exomedia.ui.widget.VideoControls in <init> at line 166
> com.devbrackets.android.exomedia.ui.widget.VideoControls in setup at line 614
> android.view.View in inflate at line 17516
> android.view.LayoutInflater in inflate at line 353
> android.view.LayoutInflater in inflate at line 397
> android.view.LayoutInflater in inflate at line 492
> android.view.LayoutInflater in rInflate at line 758
> android.view.LayoutInflater in rInflate at line 758
> android.view.LayoutInflater in rInflate at line 755
> android.view.LayoutInflater in createViewFromTag at line 694
> android.view.LayoutInflater in onCreateView at line 669
> com.android.internal.policy.impl.PhoneLayoutInflater in onCreateView at line 56
> android.view.LayoutInflater in createView at line 594
> java.lang.reflect.Constructor in newInstance at line 423
> java.lang.reflect.Constructor in constructNative
> Caused by: java.lang.reflect.InvocationTargetException
> dalvik.system.NativeStart in main
> com.android.internal.os.ZygoteInit in main at line 608
> com.android.internal.os.ZygoteInit$MethodAndArgsCaller in run at line 792
> java.lang.reflect.Method in invoke at line 515
> java.lang.reflect.Method in invokeNative
> android.app.ActivityThread in main at line 5105
> android.os.Looper in loop at line 136
> android.os.Handler in dispatchMessage at line 102
> android.app.ActivityThread$H in handleMessage at line 1200
> android.app.ActivityThread in access$800 at line 139
> android.app.ActivityThread in handleLaunchActivity at line 2250
> android.app.ActivityThread in performLaunchActivity at line 2164
> android.app.Instrumentation in callActivityOnCreate at line 1087
> android.app.Activity in performCreate at line 5275
> com.milad.appido.ExoMedia.VideoViewActitivity in onCreate at line 55
> android.app.Activity in setContentView at line 1973
> com.android.internal.policy.impl.PhoneWindow in setContentView at line 315
> android.view.LayoutInflater in inflate at line 353
> android.view.LayoutInflater in inflate at line 397
> android.view.LayoutInflater in inflate at line 492
> android.view.LayoutInflater in rInflate at line 745
> android.view.LayoutInflater in parseInclude at line 839
> android.view.LayoutInflater in rInflate at line 755
> android.view.LayoutInflater in createViewFromTag at line 696
> android.view.LayoutInflater in createView at line 594
> java.lang.reflect.Constructor in newInstance at line 423
> java.lang.reflect.Constructor in constructNative
> com.devbrackets.android.exomedia.ui.widget.EMVideoView in <init> at line 100
> com.devbrackets.android.exomedia.ui.widget.EMVideoView in setup at line 614
> com.devbrackets.android.exomedia.ui.widget.EMVideoView in readAttributes at line 636
> com.devbrackets.android.exomedia.ui.widget.VideoControlsMobile in <init> at line 49
> com.devbrackets.android.exomedia.ui.widget.VideoControls in <init> at line 166
> com.devbrackets.android.exomedia.ui.widget.VideoControls in setup at line 614
> android.view.View in inflate at line 17516
> android.view.LayoutInflater in inflate at line 353
> android.view.LayoutInflater in inflate at line 397
> android.view.LayoutInflater in inflate at line 492
> android.view.LayoutInflater in rInflate at line 758
> android.view.LayoutInflater in rInflate at line 758
> android.view.LayoutInflater in rInflate at line 755
> android.view.LayoutInflater in createViewFromTag at line 694
> android.view.LayoutInflater in onCreateView at line 669
> com.android.internal.policy.impl.PhoneLayoutInflater in onCreateView at line 56
> android.view.LayoutInflater in createView at line 620
> Caused by: android.view.InflateException ("Binary XML file line #112: Error inflating class android.widget.ImageButton")
> dalvik.system.NativeStart in main
> com.android.internal.os.ZygoteInit in main at line 608
> com.android.internal.os.ZygoteInit$MethodAndArgsCaller in run at line 792
> java.lang.reflect.Method in invoke at line 515
> java.lang.reflect.Method in invokeNative
> android.app.ActivityThread in main at line 5105
> android.os.Looper in loop at line 136
> android.os.Handler in dispatchMessage at line 102
> android.app.ActivityThread$H in handleMessage at line 1200
> android.app.ActivityThread in access$800 at line 139
> android.app.ActivityThread in handleLaunchActivity at line 2250
> android.app.ActivityThread in performLaunchActivity at line 2164
> android.app.Instrumentation in callActivityOnCreate at line 1087
> android.app.Activity in performCreate at line 5275
> com.milad.appido.ExoMedia.VideoViewActitivity in onCreate at line 55
> android.app.Activity in setContentView at line 1973
> com.android.internal.policy.impl.PhoneWindow in setContentView at line 315
> android.view.LayoutInflater in inflate at line 353
> android.view.LayoutInflater in inflate at line 397
> android.view.LayoutInflater in inflate at line 492
> android.view.LayoutInflater in rInflate at line 745
> android.view.LayoutInflater in parseInclude at line 839
> android.view.LayoutInflater in rInflate at line 755
> android.view.LayoutInflater in createViewFromTag at line 696
> android.view.LayoutInflater in createView at line 594
> java.lang.reflect.Constructor in newInstance at line 423
> java.lang.reflect.Constructor in constructNative
> Caused by: java.lang.reflect.InvocationTargetException
> dalvik.system.NativeStart in main
> com.android.internal.os.ZygoteInit in main at line 608
> com.android.internal.os.ZygoteInit$MethodAndArgsCaller in run at line 792
> java.lang.reflect.Method in invoke at line 515
> java.lang.reflect.Method in invokeNative
> android.app.ActivityThread in main at line 5105
> android.os.Looper in loop at line 136
> android.os.Handler in dispatchMessage at line 102
> android.app.ActivityThread$H in handleMessage at line 1200
> android.app.ActivityThread in access$800 at line 139
> android.app.ActivityThread in handleLaunchActivity at line 2250
> android.app.ActivityThread in performLaunchActivity at line 2164
> android.app.Instrumentation in callActivityOnCreate at line 1087
> android.app.Activity in performCreate at line 5275
> com.milad.appido.ExoMedia.VideoViewActitivity in onCreate at line 55
> android.app.Activity in setContentView at line 1973
> com.android.internal.policy.impl.PhoneWindow in setContentView at line 315
> android.view.LayoutInflater in inflate at line 353
> android.view.LayoutInflater in inflate at line 397
> android.view.LayoutInflater in inflate at line 492
> android.view.LayoutInflater in rInflate at line 745
> android.view.LayoutInflater in parseInclude at line 839
> android.view.LayoutInflater in rInflate at line 755
> android.view.LayoutInflater in createViewFromTag at line 696
> android.view.LayoutInflater in createView at line 620
> Caused by: android.view.InflateException ("Binary XML file line #19: Error inflating class com.devbrackets.android.exomedia.ui.widget.EMVideoView")
> dalvik.system.NativeStart in main
> com.android.internal.os.ZygoteInit in main at line 608
> com.android.internal.os.ZygoteInit$MethodAndArgsCaller in run at line 792
> java.lang.reflect.Method in invoke at line 515
> java.lang.reflect.Method in invokeNative
> android.app.ActivityThread in main at line 5105
> android.os.Looper in loop at line 136
> android.os.Handler in dispatchMessage at line 102
> android.app.ActivityThread$H in handleMessage at line 1200
> android.app.ActivityThread in access$800 at line 139
> android.app.ActivityThread in handleLaunchActivity at line 2250
> android.app.ActivityThread in performLaunchActivity at line 2200
> Caused by: java.lang.RuntimeException ("Unable to start activity ComponentInfo{com.rashavas.appido.v069.release/com.milad.appido.ExoMedia.VideoViewActitivity}: android.view.InflateException: Binary XML file line #19: Error inflating class com.devbracket...

--- a/demo/src/main/res/layout/audio_player_activity.xml
+++ b/demo/src/main/res/layout/audio_player_activity.xml
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
@@ -30,9 +31,9 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_alignParentBottom="true"
+        android:elevation="4dp"
         android:orientation="vertical"
         android:paddingTop="10dp"
-        android:elevation="4dp"
         android:translationZ="2dp"
         tools:background="@color/background_material_light">
 
@@ -45,20 +46,20 @@
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_alignParentLeft="true"
-                android:textColor="@android:color/black"
                 android:layout_marginLeft="10dp"
-                tools:text="00:00"
-                tools:ignore="RtlHardcoded"/>
+                android:textColor="@android:color/black"
+                tools:ignore="RtlHardcoded"
+                tools:text="00:00"/>
 
             <TextView
                 android:id="@+id/audio_player_duration"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_alignParentRight="true"
-                android:textColor="@android:color/black"
                 android:layout_marginRight="10dp"
-                tools:text="00:00"
-                tools:ignore="RelativeOverlap,RtlHardcoded"/>
+                android:textColor="@android:color/black"
+                tools:ignore="RelativeOverlap,RtlHardcoded"
+                tools:text="00:00"/>
         </RelativeLayout>
 
         <SeekBar
@@ -66,63 +67,63 @@
             style="?android:seekBarStyle"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"/>
-        
+
         <LinearLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:orientation="horizontal"
-            android:gravity="center_horizontal"
             android:layout_marginBottom="30dp"
-            android:layout_marginTop="10dp">
+            android:layout_marginTop="10dp"
+            android:gravity="center_horizontal"
+            android:orientation="horizontal">
 
             <android.support.v4.widget.Space
                 android:layout_width="0dp"
                 android:layout_height="match_parent"
-                android:layout_weight="1" />
+                android:layout_weight="1"/>
 
             <ImageButton
                 android:id="@+id/audio_player_previous"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:background="@android:color/transparent"
-                android:src="@drawable/playlistcore_ic_skip_previous_black"
                 android:scaleType="centerInside"
                 android:visibility="invisible"
-                tools:visibility="visible"
-                tools:ignore="ContentDescription"/>
+                app:srcCompat="@drawable/playlistcore_ic_skip_previous_black"
+                tools:ignore="ContentDescription"
+                tools:visibility="visible"/>
 
             <android.support.v4.widget.Space
                 android:layout_width="44dp"
-                android:layout_height="match_parent" />
+                android:layout_height="match_parent"/>
 
             <ImageButton
                 android:id="@+id/audio_player_play_pause"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:background="@android:color/transparent"
-                android:src="@drawable/playlistcore_ic_play_arrow_black"
                 android:scaleType="centerInside"
+                app:srcCompat="@drawable/playlistcore_ic_play_arrow_black"
                 tools:ignore="ContentDescription"/>
 
             <android.support.v4.widget.Space
                 android:layout_width="44dp"
-                android:layout_height="match_parent" />
+                android:layout_height="match_parent"/>
 
             <ImageButton
                 android:id="@+id/audio_player_next"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:background="@android:color/transparent"
-                android:src="@drawable/playlistcore_ic_skip_next_black"
                 android:scaleType="centerInside"
                 android:visibility="invisible"
-                tools:visibility="visible"
-                tools:ignore="ContentDescription"/>
+                app:srcCompat="@drawable/playlistcore_ic_skip_next_black"
+                tools:ignore="ContentDescription"
+                tools:visibility="visible"/>
 
             <android.support.v4.widget.Space
                 android:layout_width="0dp"
                 android:layout_height="match_parent"
-                android:layout_weight="1" />
+                android:layout_weight="1"/>
 
         </LinearLayout>
     </LinearLayout>

--- a/library/build.gradle
+++ b/library/build.gradle
@@ -32,6 +32,9 @@ android {
 
         versionCode versionMajor * 10000 + versionMinor * 1000 + versionPatch * 100
         versionName "${versionMajor}.${versionMinor}.${versionPatch}"
+
+        vectorDrawables.useSupportLibrary = true
+
         testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
     }
 

--- a/library/src/main/java/com/devbrackets/android/exomedia/util/EMResourceUtil.java
+++ b/library/src/main/java/com/devbrackets/android/exomedia/util/EMResourceUtil.java
@@ -11,6 +11,7 @@ import android.support.annotation.ColorInt;
 import android.support.annotation.ColorRes;
 import android.support.annotation.DrawableRes;
 import android.support.v4.graphics.drawable.DrawableCompat;
+import android.support.v7.widget.AppCompatDrawableManager;
 import android.util.TypedValue;
 
 /**
@@ -97,8 +98,7 @@ public class EMResourceUtil {
             return context.getResources().getDrawable(drawableResourceId, context.getTheme());
         }
 
-        //noinspection deprecation
-        return context.getResources().getDrawable(drawableResourceId);
+        return AppCompatDrawableManager.get().getDrawable(context, drawableResourceId);
     }
 
     /**

--- a/library/src/main/res/layout/exomedia_default_controls_mobile.xml
+++ b/library/src/main/res/layout/exomedia_default_controls_mobile.xml
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <com.devbrackets.android.exomedia.ui.widget.FitsSystemWindowRelativeLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent">
@@ -117,8 +118,8 @@
                 android:background="@android:color/transparent"
                 android:minHeight="@dimen/exomedia_min_button_height"
                 android:minWidth="@dimen/exomedia_min_button_height"
-                android:src="@drawable/exomedia_ic_skip_previous_white"
                 android:visibility="gone"
+                app:srcCompat="@drawable/exomedia_ic_skip_previous_white"
                 tools:ignore="ContentDescription,RtlHardcoded"
                 tools:visibility="visible"/>
 
@@ -129,7 +130,7 @@
                 android:background="@android:color/transparent"
                 android:minHeight="@dimen/exomedia_min_button_height"
                 android:minWidth="@dimen/exomedia_min_button_height"
-                android:src="@drawable/exomedia_ic_play_arrow_white"
+                app:srcCompat="@drawable/exomedia_ic_play_arrow_white"
                 tools:ignore="ContentDescription"/>
 
             <ImageButton
@@ -140,8 +141,8 @@
                 android:background="@android:color/transparent"
                 android:minHeight="@dimen/exomedia_min_button_height"
                 android:minWidth="@dimen/exomedia_min_button_height"
-                android:src="@drawable/exomedia_ic_skip_next_white"
                 android:visibility="gone"
+                app:srcCompat="@drawable/exomedia_ic_skip_next_white"
                 tools:ignore="ContentDescription,RtlHardcoded"
                 tools:visibility="visible"/>
         </LinearLayout>

